src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/RequestFeeRemissionAipPreparer.java

```diff
@@ -107,9 +93,11 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
                     asylumCase.write(PREVIOUS_REMISSION_DETAILS, remissionDetailsAppender.getRemissions());
                     assignLateRemissionValuesToRemissionValues(asylumCase);
                     clearPreviousAndLateRemissionFields(asylumCase);
+                    asylumCase.write(IS_LATE_REMISSION_REQUEST, YesOrNo.YES);
```

Comment:
What is the reason for this flag?

Reply:
Purpose of this flag as discussed with Ivan is to differentiate in the notification handler whether a record remission decision is coming from standard REMISSION or REFUND request (where refund means that is a late remission request AFTER appeal has been submitted) @ThomasKKC